frontend/src/scenes/sceneTypes.ts

```diff
@@ -53,6 +53,7 @@ export enum Scene {
     Ingestion = 'IngestionWizard',
     OnboardingSetup = 'OnboardingSetup',
     Personalization = 'Personalization',
+    APM = 'PerformanceWaterfall',
```

Comment:
Why not `APM = 'APM'`

Reply:
Hmmm 🤔  Which makes me wonder whether we could detect scenes by convention...

I'll follow-up with a rename rather than brave the flaky tests for that change :)